fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2062,6 +2062,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long max_automatic_partition_number = 4096;
 
+    /**
+     * Used to limit num of partition for load open partition number
+     */
+    @ConfField(mutable = true)
+    public static long max_load_initial_open_partition_number = 32;
```

Comment:
Why should some partitions initially opened for data loading in an expression partitioned table.

Reply:
For most scenarios, data is often only imported into the nearest partition. Opening it in advance can avoid additional RPC requests at runtime.